app/controllers/application_controller.rb

```diff
@@ -1,2 +1,9 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name nickname])
```

Comment:
Shouldn't be here `email` and `password`?

Reply:
Email and password are devise fields, they are permitted by default